mautic/mautic
# [UX] 13 new core email layout themes

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description

This PR adds 13 email themes focused on providing ready-to-use start templates.
It's a result from the [debate opened in the community portal](https://community.mautic.org/assemblies/product-team/f/39/debates/9), where I highlight how needed those themes are to improve the adoption of Mautic as a reliable tool when compared to others in the market.

The goal here is not to present a WOW design, but to have some simple layouts available to customize a bit and send. 
Because of this thought, they're built in gray tones, no colors, using placeholders, responsive, fully editable, dry code, featuring main tokens (like webview url and unsubscribe text).

I'm using the character _ to make all of them become first, leaving future designed themes to be shown after those (already existing themes are also included in this criteria).

Also implemented a folder called `@defaults` so all themes can inherit placeholders, and more people can use them to create future themes if wanted.

they also have an unsubscribe page that shows different icons based on the current action instead of just a blank screen.

![image](https://github.com/user-attachments/assets/1f0be829-77ae-4b52-817e-2976ab772a21)


![image](https://github.com/user-attachments/assets/66392f6c-b8d7-4a7b-bed2-7cdbaa898191)

![image](https://github.com/user-attachments/assets/5aa766f4-777c-418c-a3c5-29c71f33db0b)


Unsubscribed
![image](https://github.com/user-attachments/assets/e37a2f37-e195-4882-a5b8-55b05b48ceb5)

Resubscribed
![image](https://github.com/user-attachments/assets/96373da6-dd69-4919-b956-8269e94dfd8c)

Preference center (unstyled, only added the icon)
![image](https://github.com/user-attachments/assets/96d0c16f-1be4-499b-aa07-f96b93306b8b)







---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Channels > Emails and play with the theme options
